src/app/components/navbar/navbar.component.html

```diff
@@ -29,7 +29,7 @@
 
     <mat-menu #menu="matMenu">
       <button mat-menu-item (click) = "mapWindowsToDesktop()">Map windows</button>
-      <button mat-menu-item (click) = "deleteBoard()">Delete</button>
+      <button mat-menu-item (click) = "boardSettings()">Board Settings</button>
```

Comment:
"Edit Board"?

Reply:
Going to leave as settings as mention above.